typescript/framework-extensions/vercel-ai-sdk/README.md

```diff
@@ -7,25 +7,25 @@ This package is an extension used to easily plug [AgentKit](https://docs.cdp.coi
 For a single command to install all necessary dependencies, run:
 
 ```bash
-npm install @coinbase/agentkit-vercel-ai-sdk @coinbase/agentkit ai @ai-sdk/openai
+pnpm add @coinbase/agentkit-vercel-ai-sdk @coinbase/agentkit ai @ai-sdk/openai
```

Comment:
I'm wondering if we should leave user-facing commands as `npm`. Contributors to AgentKit need `pnpm`, consumers don't

Reply:
Yeah, I think you're right